resources/views/pages/blocks/widgets/photos.blade.php

```diff
@@ -1,60 +1,28 @@
 <div class="ui-block">
     <div class="ui-block-title">
-        <h6 class="title">Last Photos</h6>
+        <h6 class="title">@lang('Last Photos')</h6>
     </div>
     <div class="ui-block-content">
 
         <!-- W-Latest-Photo -->
 
         <ul class="widget w-last-photo js-zoom-gallery">
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-photo10-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-photo10-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot11-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot11-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot12-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot12-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot13-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot13-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot14-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot14-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot15-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot15-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot16-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot16-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot17-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot17-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-            <li>
-                <a href="{{ asset('theme/socialyte/img/last-phot18-large.jpg') }}">
-                    <img src="{{ asset('theme/socialyte/img/last-phot18-large.jpg') }}" alt="photo">
-                </a>
-            </li>
-        </ul>
-
+            @foreach ($posts as $post)
+                @php
+                    $images = json_decode($post->image);
+                @endphp
 
+                @if (is_array($images) || is_object($images))
+                    @foreach ($images as $postImage)
```

Comment:
```suggestion
                    @foreach (json_decode($post->image) as $postImage)
```
e thử thế này luôn xem được ko, ko cần check if đâu